README.md

```diff
@@ -555,43 +557,45 @@ Log files are separated out for each context (assuming `context-per-jvm` is `tru
 Note: to test out the deploy to a local staging dir, or package the job server for Mesos,
 use `bin/server_package.sh <environment>`.
 
-#### Configuring Spark Jobserver meta data Database backend
-
+#### Configuring Spark Jobserver H2 Database backend
 By default, H2 database is used for storing Spark Jobserver related meta data.
-But this can be overridden. For example, to use PostgreSQL as backend add the
-following configuration to local.conf. Ensure that you have spark_jobserver
-database created with necessary rights granted to user.
+This can be overridden if you prefer to use PostgreSQL or MySQL.
+It is also important that any dependent jars are to be added to Job Server class path.
 
-    sqldao {
-      # Slick database driver, full classpath
-      slick-driver = slick.driver.PostgresDriver
+To use embedded H2 as backend add the following configuration to local.conf.
 
-      # JDBC driver, full classpath
-      jdbc-driver = org.postgresql.Driver
+    spark {
+      jobserver {
+        ...
+        sqldao {
+          # Slick database driver, full classpath
+          slick-driver = slick.driver.H2Driver
 
-      # Directory where default H2 driver stores its data. Only needed for H2.
-      rootdir = "/var/spark-jobserver/sqldao/data"
+          # JDBC driver, full classpath
+          jdbc-driver = org.h2.Driver
 
-      jdbc {
-        url = "jdbc:postgresql://db_host/spark_jobserver"
-        user = "secret"
-        password = "secret"
-      }
+          # Directory where default H2 driver stores its data. Only needed for H2.
+          rootdir = "/var/spark-jobserver/sqldao/data"
+
+          jdbc {
```

Comment:
I wonder if the database config part of doc is getting too big and we should move them to doc/.  I suppose that can be a separate PR.